code/data_preprocessing/reference_data.ipynb

```diff
@@ -782,13 +782,44 @@
     "        f\"{cwd}/RSEM_Index/rsem_reference.chrlist\", f\"{cwd}/RSEM_Index/rsem_reference.seq\", \n",
     "        f\"{cwd}/RSEM_Index/rsem_reference.transcripts.fa\"\n",
     "task: trunk_workers = 1, trunk_size = job_size, walltime = walltime, mem = mem, tags = f'{step_name}_{_output[0]:bd}'\n",
-    "bash: container=container, expand= \"${ }\", stderr = f'{_output[0]:d}.stderr', stdout = f'{_output[0]:d}.stdout'\n",
+    "bash: container=container, expand= \"${ }\", stderr = f'{_output[0]}.stderr', stdout = f'{_output[0]}.stdout'\n",
     "    rsem-prepare-reference \\\n",
     "            ${_input[0]} \\\n",
     "            ${_output[1]:n} \\\n",
     "            --gtf ${_input[1]} \\\n",
     "            --num-threads ${numThreads}"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e8d4e2f-0138-4dc1-832e-301f23d1bb73",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "source": [
+    "## Generation of RefFlat file \n",
+    "This file is needed for picard CollectRnaSeqMetrics module, which in turn \n",
+    ">produces metrics describing the distribution of the bases within the transcripts. It calculates the total numbers and the fractions of nucleotides within specific genomic regions including untranslated regions (UTRs), introns, intergenic sequences (between discrete genes), and peptide-coding sequences (exons). This tool also determines the numbers of bases that pass quality filters that are specific to Illumina data (PF_BASES)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "87d04978-b7c8-4873-a192-d4d4e8c3d0a1",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "[RefFlat_generation]\n",
```

Comment:
is this the step QC we ended up not using later? If so, should we still keep it?

Reply:
It is for the picard qc